.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VS Code settings
+.vscode/
```

Comment:
This file shouldn't be in the PR. I guess you can configure your IDE to ignore it